qemu/tests/cfg/check_link_speed_duplex.cfg

```diff
@@ -0,0 +1,16 @@
+- check_link_speed_duplex:
+    image_snapshot = yes
+    type = check_link_speed_duplex
+    tar_duplex = "full"
+    modify_speed_param = ",speed=1000,duplex=full"
+    Windows:
+        cdroms += " virtio"
+        check_speed_powershell_cmd = 'powershell -command "GET-NETADAPTER | SELECT SPEED, FULLDUPLEX"'
+        Win7, Win2008, Win2008..r2:
+            run_powershell = no
+        default_tar_speed = 100000000000
+        modify_tar_speed = 1000000000
+    Linux:
+        check_speed_cmd = "ethtool %s"
+        default_tar_speed = 100000
+        modify_tar_speed = 1000
```

Comment:
`modify_speed_param` better be `",speed=${modify_tar_speed},duplex=${tar_duplex}"`
and windows value seems would be *1000000, that could be handled in code, better than manually specified here.